fb-install-foreman.bats

```diff
@@ -161,7 +161,7 @@ EOF
 }
 
 @test "wake up puppet agent" {
-  puppet agent -t -v
```

Comment:
Sorry what to keep? I was under impression that `-o --no-daemonize` == `-t` therefore no change.